src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/pureThermalDiffusion/Example.rst

```diff
@@ -0,0 +1,105 @@
+.. _AdvancedExamplePureThermalDiffusionWellbore:
+
+
+####################################################
+Pure thermal diffusion around a wellbore
+####################################################
+
+------------------------------------------------------------------
+Problem description
+------------------------------------------------------------------
+
+This example uses the thermal flow solver to a pure thermal diffusion problem around a wellbore. Thermal convection and fluid flow are removed by settting fluid pressure and fluid heat capacity to zero to mimic a pure thermal diffusion problem, .
+
```

Comment:
Adding a figure here could help to describe the settings of the wellbore problem.

Reply:
done